### ISSUE TITLE
Cannot copy/paste using buttons in folder_contents page

### ISSUE THREAD

**Issue by ebrehault** [plone/Products.CMFPlone#645]
- go to folder1, click Contents in the toolbar
- select an element and click on the Copy button,
- go to folder 2, click Contents in the toolbar,
- the Paste button is disabled,
- the Paste action is not available in the toolbar's More options menu.

Similarly:
- go to doc1, and in toolbar's More options menu, click Copy,
- go to folder 2, click Contents in the toolbar,
- the Paste button is disabled,
- but in View mode, the Paste action is available in the toolbar's More options menu.

And if we just use the toolbar copy and paste actions, it just works.

So basically, the problem seems to be about the copy and paste buttons in the folder_contents page.

**Comment by vangheem** [comment]
The reason this is like this is the default manage_cutObjects and manage_copyObjects do not work with objects that are in different folders whereas in folder contents, you can gather objects in multiple folders and then paste them.

However, I'm pretty sure it can still be done using the same machinery and then paying attention to the cookies being set.

Thanks for bringing this up. It's something that got forgotten.

**Comment by vangheem** [comment]
Fixed with https://github.com/plone/plone.app.content/commit/336a39447cda97fe78c3cf6f60861ea7631b60f7
